openlibrary/catalog/marc/parse.py

```diff
@@ -32,6 +32,23 @@
 re_bracket_field = re.compile(r'^\s*(\[.*\])\.?\s*$')
 
 
+ROLES = {
+    # MARC 21 relator codes
+    # https://www.loc.gov/marc/relators/relacode.html
+    'art': 'Artist',
+    'aut': 'Author',
+    'clr': 'Colorist',
+    'com': 'Compiler',
+    'edt': 'Editor',
+    'ill': 'Illustrator',
+    'ltr': 'Letterer',
+    'trl': 'Translator',
+    # Non-standard terms from $e
+    'ed.': 'Editor',
+    'comp.': 'Compiler',
```

Comment:
```suggestion
    'comp.': 'Compiler',
    'ed.': 'Editor',
```
Alphabetized for a combination of pedantry and ease of scanning ;)